contracts/NestedFactory.sol

```diff
@@ -439,11 +426,8 @@ contract NestedFactory is INestedFactory, ReentrancyGuard, OwnableProxyDelegatio
         amountBought = _batchedOrders.outputToken.balanceOf(address(this)) - amountBought;
 
         unchecked {
```

Comment:
Since there is no more math in this `unchecked` block, it can be deleted.

Reply:
Indeed !
Fixed